selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -264,8 +264,8 @@ void SoftwarePanel::updateLabels() {
   updateBtn->setText("CHECK");
   updateBtn->setEnabled(true);
   gitBranchLbl->setText(QString::fromStdString(params.get("GitBranch")));
-  gitCommitLbl->setText(QString::fromStdString(params.get("GitCommit")).left(10));
-  osVersionLbl->setText(QString::fromStdString(Hardware::get_os_version()).trimmed());
+  gitCommitLbl->setText(QString::fromStdString(params.get("GitCommit")));
+  osVersionLbl->setText(QString::fromStdString(Hardware::get_os_version()));
```

Comment:
short hash is better than cutoff long one
```suggestion
  gitCommitLbl->setText(QString::fromStdString(params.get("GitCommit")).left(10));
  osVersionLbl->setText(QString::fromStdString(Hardware::get_os_version()).trimmed());
```